addon/components/step-manager.js

```diff
@@ -242,11 +242,9 @@ export default Component.extend({
      * @public
      */
     'transition-to-previous-step'(value) {
-      const to = get(this, '_lastStep');
```

Comment:
Definitely prefer moving this to the state machine, rather than having the component remember 👍